api/envoy/service/auth/v2/attribute_context.proto

```diff
@@ -67,9 +67,10 @@ message AttributeContext {
     // `URI Subject Alternative Names`, `DNS Subject Alternate Names` or `Subject` in that order.
     // The primary identity should be the principal. The principal format is issuer specific.
     //
-    // Example:
-    // *    SPIFFE format is `spiffe://trust-domain/path`
-    // *    Google account format is `https://accounts.google.com/{userid}`
+    // Examples:
+    //
+    // - SPIFFE format is `spiffe://trust-domain/path`.
```

Comment:
not sure the spiffe: link should be a literal - but if so it should use double backticks ` `` `

Reply:
I see, we should use double backticks for v3.

Reply:
its more just how it is rendered from rst - we previously had a policy doc that said use single backticks - but we changed that a long time ago

better leave the v2 protos alone tho - yep